auto-random.md

```diff
@@ -12,147 +12,108 @@ aliases: ['/docs/dev/auto-random/','/docs/dev/reference/sql/attributes/auto-rand
-
-To use different number of shard bits, append a pair of parentheses to `AUTO_RANDOM` and specify the desired number of shard bits in the parentheses. See the following example:
-
-{{< copyable "sql" >}}
+- If you explicitly specify the value of the `AUTO_RANDOM` column, the value you specified is inserted into the table.
+- If you do not explicitly specify the value of the `AUTO_RANDOM` column, TiDB generates a random value and inserts it into the table.
 
 ```sql
-CREATE TABLE t (a bigint PRIMARY KEY AUTO_RANDOM(3), b varchar(255))
+tidb> CREATE TABLE t (a BIGINT PRIMARY KEY AUTO_RANDOM, b VARCHAR(255));
+Query OK, 0 rows affected, 1 warning (0.01 sec)
+
+tidb> INSERT INTO t(a, b) VALUES (1, 'string');
+Query OK, 1 row affected (0.00 sec)
+
+tidb> SELECT * FROM t;
++---+--------+
+| a | b      |
++---+--------+
+| 1 | string |
++---+--------+
+1 row in set (0.01 sec)
+
+mysql> INSERT INTO t(b) VALUES ('string2');
+Query OK, 1 row affected (0.00 sec)
+
+mysql> INSERT INTO t(b) VALUES ('string3');
+Query OK, 1 row affected (0.00 sec)
+
+mysql> SELECT * FROM t;
++---------------------+---------+
+| a                   | b       |
++---------------------+---------+
+|                   1 | string  |
+| 1152921504606846978 | string2 |
+| 4899916394579099651 | string3 |
++---------------------+---------+
+3 rows in set (0.00 sec)
 ```
 
-In the above `CREATE TABLE` statement, `3` shard bits are specified. The range of the number of shard bits is `[1, 16)`.
+The `AUTO_RANDOM(S, R)` column value automatically assigned by TiDB has a total of 64 bits:
 
-After creating the table, use the `SHOW WARNINGS` statement to see the maximum number of implicit allocations supported by the current table:
+- `S` is the number of shard bits. The value range is from `1` to `15`. The default value is `5`.
+- `R` is the total length of the automatic allocation range. The value range is from `32` to `64`. The default value is `64`.
 
-{{< copyable "sql" >}}
+The value structure of an `AUTO_RANDOM` column is as follows:
 
-```sql
-SHOW WARNINGS
-```
+| Total number of bits | Sign bit | Reserved bit | Shard bit | Auto-increment bit |
+|---------|--------|-------------|--------|--------------|
+| 64 bits | 0/1 bit | (64-R) bits | S bits | (R-1-S) bits |
 
-```sql
-+-------+------+----------------------------------------------------------+
-| Level | Code | Message                                                  |
-+-------+------+----------------------------------------------------------+
-| Note  | 1105 | Available implicit allocation times: 1152921504606846976 |
-+-------+------+----------------------------------------------------------+
-```
+- The length of the sign bit is determined by whether an `UNSIGNED` attribute exists. If there is an `UNSIGNED` attribute, the sign bit is `0`. Otherwise, the sign bit is `1`.
```

Comment:
```suggestion
- The the sign bit is determined by whether an `UNSIGNED` attribute exists. If there is an `UNSIGNED` attribute, the length of it is `0`. Otherwise, the length of it is `1`.
```